### ISSUE TITLE
Error when try to re-generate certificate

### ISSUE THREAD

**Issue by carloscds** [sjkp/letsencrypt-siteextension#294]
System.NullReferenceException: Object reference not set to an instance of an object. at LetsEncrypt.SiteExtension.Controllers.HomeController.<Index>d__1.MoveNext() in D:\a\1\s\LetsEncrypt-SiteExtension\Controllers\HomeController.cs:line 0

**Comment by sjkp** [comment]
Are you on latest version?

**Comment by carloscds** [comment]
@carloscds Yes, after some tests I discover that subscription ID was wrong.

**Comment by sjkp** [comment]
Okay, will look into the error message. Can probably be handled better.

**Comment by carloscds** [comment]
@sjkp Thanks!
